app/03app_node/board.h

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file
+ * @brief       Generic board support interface.
+ * @details
+ *
+ */
+#ifndef BOARD_H
+#define BOARD_H
+
+typedef enum {
+    OFF,
+    RED,
+    GREEN,
+    BLUE,
+    OTHER,
+} led_color_t;
+
+void board_init(void);
+void board_set_rgb(led_color_t color);
```

Comment:
I think we should 

1. rename this to `board_set_led_mira_status`
2. add a function `board_set_led_app`. in the minimote it does nothing, and in the dongle it controls the other LED.